src/ui/common/src/components/integrations/addIntegrations.tsx

```diff
@@ -45,7 +45,7 @@ const AddIntegrations: React.FC<Props> = ({
       )}
       <Box sx={{ width: '100%', display: 'flex', flexWrap: 'wrap' }}>
         {Object.entries(supportedIntegrations)
-          .filter(([svc]) => svc !== 'Aqueduct Demo')
+          .filter(([svc]) => svc !== 'Aqueduct')
```

Comment:
We should probably have these in an enum in the future.

Filed a small linear task for this work here:
https://linear.app/aqueducthq/issue/ENG-2836/ui-put-integration-names-into-an-enum